pkgs/games/sgt-puzzles/enable-mobile-ui.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 9e8d96f..3e333b1 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -1,5 +1,7 @@
+ cmake_minimum_required(VERSION 3.5)
+ 
++add_compile_definitions(STYLUS_BASED)
```

Comment:
Is this the same as setting cflags / CMAKE_C_FLAGS? Changing the cmake file seems a bit more brittle.

Reply:
Have you tried `NIX_CFLAGS_COMPILE` ? Maybe hacky, but has a pretty good chance of working no matter the build system…